extensions/ql-vscode/test/pure-tests/databases/db-config-store.test.ts

```diff
@@ -0,0 +1,57 @@
+import * as fs from 'fs-extra';
+import * as path from 'path';
+import { DbConfigStore } from '../../../src/databases/db-config-store';
+import { expect } from 'chai';
+
+
+describe('db config store', async () => {
+  const workspaceStoragePath = path.join(__dirname, 'test-workspace');
+  const testStoragePath = path.join(__dirname, 'data');
```

Comment:
Should we use a copy of this for every test that we can remove in an afterEach, so that tests cannot influence each other?

Reply:
There are two dirs - I've updated the two variables to hopefully make more sense!